src/frontend/apps/e2e/__tests__/app-desk/mail-domains.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { expect, test } from '@playwright/test';
+import { MailDomain } from 'app-desk/src/features/mail-domains';
+
+import { keyCloakSignIn } from './common';
+
+const API_ROUTE_MAIL_DOMAINS = `*/**/api/v1.0/mail-domains/`;
+
+const currentDateIso = new Date().toISOString();
+const mailDomainsFixtures: MailDomain[] = [
+  {
+    name: 'domain.fr',
+    id: '456ac6ca-0402-4615-8005-69bc1efde43f',
+    created_at: currentDateIso,
+    updated_at: currentDateIso,
+  },
+  {
+    name: 'mails.fr',
+    id: '456ac6ca-0402-4615-8005-69bc1efde43e',
+    created_at: currentDateIso,
+    updated_at: currentDateIso,
+  },
+  {
+    name: 'versailles.net',
+    id: '456ac6ca-0402-4615-8005-69bc1efde43g',
+    created_at: currentDateIso,
+    updated_at: currentDateIso,
+  },
+  {
+    name: 'paris.fr',
+    id: '456ac6ca-0402-4615-8005-69bc1efde43h',
+    created_at: currentDateIso,
+    updated_at: currentDateIso,
+  },
+];
+
+test.describe('Mail domain', () => {
+  test.describe('checks all the elements are visible', () => {
+    test.beforeEach(async ({ page, browserName }) => {
+      await page.goto('/');
+      await keyCloakSignIn(page, browserName);
+    });
+
+    test('when no mail domain exists', async ({ page }) => {
+      await page.route(`${API_ROUTE_MAIL_DOMAINS}**`, async (route) => {
+        await route.fulfill({
+          json: {
+            count: 0,
+            next: null,
+            previous: null,
+            results: [],
+          },
+        });
+      });
```

Comment:
For the moment it is working, but you will have quickly some problems to make it so generic. 
What about being a bit more specific ? 

```suggestion
      await page.route(`**/api/v1.0/mail-domains/?page=*`, async (route) => {
        const request = route.request();
        if (request.method().includes('GET')) {
          await route.fulfill({
          json: {
              count: 0,
              next: null,
              previous: null,
              results: [],
            },
          });
        } else {
          await route.continue();
        }
      });

```